api/server/server.go

```diff
@@ -197,11 +211,23 @@ type restBase struct {
 	name    string
 }
 
+func regxAnonString(inputStr string) string {
```

Comment:
maybe add a quick unit test for this function. To make sure the regex is working as expected

Reply:
Yeah sounds good..